app/services/DeclarationTypeService.scala

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package services
+
+import connectors.ReferenceDataConnector
+import models.{DeclarationTypeItemLevel, UserAnswers}
+import uk.gov.hmrc.http.HeaderCarrier
+
+import javax.inject.Inject
+import scala.concurrent.{ExecutionContext, Future}
+
+class DeclarationTypeService @Inject() (referenceDataConnector: ReferenceDataConnector)(implicit ec: ExecutionContext) {
```

Comment:
Do we need to filter out T2SM as per Sayak's comment in the ticket? https://github.com/hmrc/customs-reference-data/blob/main/conf/resources/DeclarationTypeItemLevel.json

Reply:
Ah ok. Yeah if we never need that value then the service layer probably more appropriate I think

Reply:
Yep, moved this filter out of the controller, and placed it in the DeclarationTypeService.